cdc/model/schema_storage.go

```diff
@@ -140,6 +172,59 @@ func (ti *TableInfo) findHandleIndex() {
 	}
 }
 
+func (ti *TableInfo) initColumnsFlag() {
+	for _, colInfo := range ti.Columns {
+		var flag ColumnFlagType
+		if colInfo.Charset == "binary" {
+			flag.SetIsBinary()
+		}
+		if colInfo.IsGenerated() {
+			flag.SetIsGeneratedColumn()
+		}
+		if mysql.HasPriKeyFlag(colInfo.Flag) {
+			flag.SetIsPrimaryKey()
+			if ti.HandleIndexID == HandleIndexPKIsHandle {
+				flag.SetIsHandleKey()
+			}
+		}
+		if mysql.HasUniKeyFlag(colInfo.Flag) {
+			flag.SetIsUniqueKey()
+		}
+		if !mysql.HasNotNullFlag(colInfo.Flag) {
+			flag.SetIsNullable()
+		}
+		if mysql.HasMultipleKeyFlag(colInfo.Flag) {
+			flag.SetIsMultipleKey()
+		}
+		ti.ColumnsFlag[colInfo.ID] = flag
+	}
+
+	// In TiDB, Only the first column can be set multiple key,
```

Comment:
Suggested:
In TiDB, just as in MySql, only the first column of an index can be marked as "multiple key" or "unique key", and only the first column of a unique index may be marked as "unique key". See https://dev.mysql.com/doc/refman/5.7/en/show-columns.html.
Yet if an index has multiple columns, we would like to easily determine that all those columns are indexed, which is crucial for the completeness of the information we pass to the downstream. Therefore, instead of using the MySql standard, we made our own decision to mark all columns in an index with the appropriate flag(s).